input/intro-notes/StructureDefinition-au-core-patient-intro.md

```diff
@@ -14,7 +14,7 @@ The following are supported usage scenarios for this profile:
 A resource conforming to this profile is conformant to:
 - [IPA-Patient](http://hl7.org/fhir/uv/ipa/StructureDefinition-ipa-patient.html)
 - [Patient (IPS)](http://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Patient-uv-ips.html)
-- [US Core Patient](http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient)
+- [US Core Patient](http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient) noting that US Core Patient includes US Gore Gender Identity extension that is not required to be supported
```

Comment:
Change US Core bit to 
if gender identity is not exchanged. US Core includes the US Gore Gender Identity extension; AU Core requires support of the international Gender Identity extension"

Reply:
Done